packages/electron-builder-squirrel-windows/src/SquirrelWindowsTarget.ts

```diff
@@ -96,39 +108,55 @@ export default class SquirrelWindowsTarget extends Target {
     checkConflictingOptions(this.options)
 
     const appInfo = packager.appInfo
-    const projectUrl = await appInfo.computePackageUrl()
-    const appName = this.appName
+    // If not specified will use the Squirrel.Windows that is shipped with electron-installer(https://github.com/electron/windows-installer/tree/main/vendor)
+    // After https://github.com/electron-userland/electron-builder-binaries/pull/56 merged, will add `electron-builder-binaries` to get the latest version of squirrel.
+    let vendorDirectory = this.options.customSquirrelExePath
```

Comment:
`customSquirrelExePath` this needs to be renamed as it doesn't correspond to the tsdoc definition. (`The custom squirrel.windows.exe path.`)

Please update to something like `customSquirrelVendorDir` or `Path` or something like that.